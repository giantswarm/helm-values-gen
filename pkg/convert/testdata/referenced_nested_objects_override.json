{
  "$schema": "https://json-schema.org/draft/2020-12/schema",
  "properties": {
    "address": {
      "$ref": "address.json",
      "type": "object",
      "properties": {
        "street": {
          "type": "string",
          "default": "Broadway"
        }
      }
    },
    "age": {
      "type": "integer",
      "default": 21
    },
    "firstName": {
      "type": "string",
      "default": "John"
    },
    "lastName": {
      "type": "string",
      "default": "Doe"
    }
  },
  "type": "object"
}
