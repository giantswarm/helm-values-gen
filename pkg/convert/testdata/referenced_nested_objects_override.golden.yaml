address:
  city: New York
  state: NY
  street: Broadway
  zip: "12345"
age: 21
firstName: John
lastName: Doe
